stories/components/MdInput.stories.tsx

```diff
@@ -7,18 +8,46 @@ export default {
   title: 'Components/MdInput'
 }
 
-export const Basic: React.FC = () => <MdInput />
+const Template: Story = (args: { presetValue?: string }) => {
+  // Mock state of parent component (ReviewCard.tsx)
```

Comment:
I'm a little confused here. The `MdInput` doesn't depend on `ReviewCard.tsx` to be used, so I think you meant just `Mock parent` or something like that, but a comment isn't necessary here because the code is very clear.

Reply:
That's a better way to put it - I will remove that comment